src/main/java/module/opponentspy/SkillAdjuster.java

```diff
@@ -34,7 +34,7 @@ public void AdjustPlayer(CalcVariables calcPlayer) {
 			if (needMoreMainSkill(calcPlayer)) {
 				// Reduce all subskills by 1. Then increase all skills until reaching wage or tsi limit.
 			
-				System.out.println("Need main");
+				// /System.out.println("Need main");
```

Comment:
delete this

Reply:
opponentspy was not my code - i only debugged it a little